Scripts/script-CommonServer.yml

```diff
@@ -98,15 +98,19 @@ script: |-
       }
 
       //in case of headers was not provided (backward compatibility)
-      if (t && (!(headers instanceof Array && headers.length > 0) || !headers)){
+      if ((!headers || !(headers instanceof Array)) && t && t.length > 0){
           headers = Object.keys(t[0]);
       }
 
+      if(!headers || !(headers instanceof Array) || headers.length === 0) {
```

Comment:
use !!headers.length and not headers.length === 0, and no need to put () around headers instanceof Array